capplets/appearance/appearance.h

```diff
@@ -58,6 +58,7 @@
 #define GTK_THEME_KEY                "gtk-theme"
 #define ICON_THEME_KEY               "icon-theme"
 #define INTERFACE_SCHEMA             "org.mate.interface"
+#define INTERFACE_GNOME_SCHEMA       "org.gnome.desktop.interface"
```

Comment:
does this add a strong dependency on GNOME desktop schemas?

Reply:
Not a new one, pretty sure gtk3 itself depends on that, correct me if I am wrong.

Reply:
Apparently it does use it for the Wayland backend indeed.  Though, if I read [the code](https://gitlab.gnome.org/GNOME/gtk/-/tree/gtk-3-24/gdk/wayland/gdkscreen-wayland.c#L714) properly it's a soft-dependency, and a fallback one at that, and it uses the DBus `org.freedesktop.portal.Settings` interface on the `org.freedesktop.portal.Desktop` session bus (plus something for the fontconfig settings).  So either that's what Wayfire provides, or we'd just have to provide that instead of an XSettings backend in MSD.

Reply:
So what I mean is that this *does* make MCC depend on the GNOME desktop schemas.  Whether another component needed for the Wayland session already does or not is yet to be determined.

Reply:
I will be adding the suggested code to look for the schema and ignore it (and wayland theming) if it is not present

Reply:
Does defining this in the header file create a hard dependency when it is not present even when we explicitly check for it before using it, or not?  If it does, we should probably move this out of the header file and just use the schema name directly.